src/hotspot/os_cpu/linux_riscv/vm_version_linux_riscv.cpp

```diff
@@ -135,3 +97,56 @@ void VM_Version::get_os_cpu_info() {
     fclose(f);
   }
 }
+
```

Comment:
Can you add a comment here about how the 'isa' substring with those extensions look like?

Reply:
Fixed. Add an example of isa string and explain how multi-letter extensions are separated.